src/components/survey/Survey.tsx

```diff
@@ -0,0 +1,90 @@
+"use client"
+import { ChangeEvent, useState, useCallback } from 'react';
+
+
+interface SurveyProps {
+  title: string;
+  lowest: string;
+  highest: string;
+}
+
+export const Survey = ({ title, lowest, highest  }: SurveyProps) => {
+  const [isOpen, setIsOpen] = useState(false);
+  const [selectedRating, setSelectedRating] = useState("0");
+
+	const handleRatingClick = (event: any) => {
+		setSelectedRating(event.target.value);
+	};
+
+  const handleSubmit = (event: any) => {
+    event.preventDefault();
+    // alert(`Rating: ${selectedRating}`);
+    console.log(`Submitted rating: ${selectedRating}`);
+  };
+
+
+  return (
+    <div className="absolute bottom-0 right-0">
+       {isOpen ? (
+        <div className='flex flex-col items-end'>      
+          <button className="bg-white rounded w-10 flex flex-col items-center" onClick={() => setIsOpen(!isOpen)}>
+            <svg 
+              xmlns="http://www.w3.org/2000/svg" 
+              fill="none" 
+              viewBox="0 0 24 24" 
+              strokeWidth={1.5} 
+              stroke="currentColor" 
+              className="w-6 h-6">
+              <path 
+                strokeLinecap="round" strokeLinejoin="round" d="M19.5 8.25l-7.5 7.5-7.5-7.5" />
+            </svg>
+          </button>
+
+          <form onSubmit={handleSubmit} className="border-4 bg-white rounded max-w-lg flex-col items-center content-center" >
+            <div className="m-2"><b>{title}</b></div>
```

Comment:
See if you can find something to prevent using `<b>`? `<b>` is semantically used to [bring attention to an element](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b), which I am not sure is desirable. If you want to just style this so it has a bold font-weight, we should look into an alternative solution.